subprojects/launcher/src/test/groovy/org/gradle/tooling/internal/provider/ContinuousBuildActionExecutorTest.groovy

```diff
@@ -86,24 +87,31 @@ class ContinuousBuildActionExecutorTest extends ConcurrentSpec {
     }
 
     def cleanup() {
+        println("Build Output:")
+        println(textOutputFactory)
```

Comment:
Do we want to keep these?

Reply:
Yeah, I added those so you see the build output in each test case. I think it helps in understanding that the test does the correct thing when you actually run them.